tailcfg/node.go

```diff
@@ -0,0 +1,105 @@
+package tailcfg
```

Comment:
It's too hard to review & it screws with future archaeology when you do moves & changes in the same commit.

Just keep this all where it was for now. We can move it later in a move-only change.